keps/1-kubernetes-enhancement-proposal-process.md

```diff
@@ -221,106 +218,34 @@ Metadata items:
-will drive decisions on what features to maintain and which to deprecate and so
-it is possible that a KEP would be retired if its functionality no longer provides
-sufficient value to the community.
+A KEP is has the following states
+
+- `Draft`: The KEP has been proposed and is actively being defined.
+- `Deferred`: The KEP is proposed but not actively being worked on.
+- `Approved`: Using lazy consensus, the approvers have approved this KEP.
+  It will move forward to implementation.
+- `Rejected`: The approvers and authors have decided that this KEP is not moving forward.
+  The KEP is kept around as a historical document.
+- `Withdrawn`: The KEP has been withdrawn by the authors.
+- `Final`: The KEP has been implemented and is no longer actively changed.
+- `Replaced`: The KEP has been replaced by a new KEP.
+  The `superseded-by` metadata value should point to the new KEP.
 
 ### Git and GitHub Implementation
 
-Practically an KEP would be implemented as a pull request to a central repository
-with the following example structure
-
-```
-├── 0000-kep-template.md
-├── CODEOWNERS
-├── index.md
-├── sig-architecture
-│   ├── deferred
-│   ├── orphaned
-│   └── retired
-├── sig-network
-│   ├── deferred
-│   ├── kube-dns
-│   ├── orphaned
-│   └── retired
-├── sig-node
-│   ├── deferred
-│   ├── kubelet
-│   ├── orphaned
-│   └── retired
-├── sig-release
-│   ├── deferred
-│   ├── orphaned
-│   └── retired
-├── sig-storage
-│   ├── deferred
-│   ├── orphaned
-│   └── retired
-├── unsorted-to-be-used-by-newcomers-only
-└── wg-resource-management
-    ├── deferred
-    ├── orphaned
-    └── retired
-```
-
-where each SIG or working group is given a top level directory with subprojects
-maintained by the SIG listed in sub directories. For newcomers to the community
-an `unsorted-to-be-used-by-newcomers-only` directory may be used before an KEP
-can be properly routed to a SIG although hopefully if discussion for a potential
-KEP begins on the mailing lists proper routing information will be provided to
-the KEP author. Additionally a top level index of KEPs may be helpful for people
-looking for a complete list of KEPs. There should be basic CI to ensure that an
-`index.md` remains up to date.
-
-Ideally no work would begin within the repositories of the Kubernetes organization
-before a KEP has been approved by the responsible SIG or working group. While the
-details of how SIGs organize their work is beyond the scope of this proposal one
-possibility would be for each charter SIG to create a top level repository within
-the Kubernetes org where implementation issues managed by that SIG would be filed.
+KEPs are checked into the community repo under the `/kep` directory.
+KEPs can either be put at the top level of that directory or in a SIG specific subdirectory.
+KEPs in SIG specific subdirectories have limited impact outside of the SIG and can leverage SIG specific OWNERS files.
```

Comment:
if this is so, then would the `kubernetes-wide` special value be needed at all? Also why then add an `owning-sig` field in the metadata?

Reply:
I'm thinking that there would be a small number of KEPs like that but they'll have a big impact.  The owning SIG would be the main driver of the change even though it can/would impact a large number of other SIGs.  This would be a shorthand for "this'll impact pretty much everyone".

Reply:
Is that not adequately conveyed by the `owning-sig` being SIG Architecture or SIG {ContribX, PM} for that matter?

Reply:
If we have some UI to say "show me all of the KEPs that impact SIG-foo" we then need a special value that means `*`.  I'm happy using `*` instead of `kubernetes-wide`.

Reply:
I don't know if a wildcard is a hard requirement, as I've experienced it (like how it works for BASH) it's nice sugar but it isn't a "must have"